backstop/post-release.json

```diff
@@ -0,0 +1,22 @@
+[
+  {"label": "Document", "url": "/media/1268726"},
+  {"label": "HomepageWithAlert", "url":  "/", "showAlerts":  true},
```

Comment:
These 2 lines will be stale after a release so checking them doesn’t tell us much. We need to add them to the post-release purge if we want to backstop them https://github.com/massgov/openmass/blob/develop/drush/Commands/DeployCommands.php#L326-L330

Reply:
Hey @weitzman , https://github.com/massgov/openmass/blob/develop/drush/Commands/DeployCommands.php#L340
This line seems to be adding homepage to the queue, correct me if I'm wrong. If that's the case I think i need to add another item to the array for media path.

Reply:
Yes you are correct. I didn’t notice that.